src/Generator/Generators/CLI/CLITypeReferences.cs

```diff
@@ -104,39 +121,27 @@ private void GenerateInclude(ASTRecord<Declaration> record)
             if (decl.Namespace == null)
                 return;
 
+            var typedefType = record.Value as TypedefNameDecl;
+
             // Find a type map for the declaration and use it if it exists.
             TypeMap typeMap;
-            if (TypeMapDatabase.FindTypeMap(record.Value, out typeMap))
+            if (TypeMapDatabase.FindTypeMap(record.Value, out typeMap) 
+                || (typedefType != null && TypeMapDatabase.FindTypeMap(typedefType.Type.Desugar(), out typeMap)))
```

Comment:
Just FYI this should be fixed inside `FindTypeMap` itself.

I do have a PR that I intend to land soon that rework type map handling and so we should be able to remove this then.

Reply:
So do you want to keep this in the commit until your PR, or remove it from this commit and wait for your PR? I would sort of prefer the former to unblock the work I'm doing if that's OK?

Reply:
Yes, that's fine to keep it.